tests/keras/backend/backend_test.py

```diff
@@ -1852,6 +1852,16 @@ def test_variable_support_bool_dtype(self):
             with pytest.raises(TypeError):
                 K.variable('', dtype='unsupported')
 
+    @pytest.mark.skipif((K.backend() != 'mxnet'),
+                        reason='Testing only for MXNet backend')
+    def test_tensor_slicing(self):
```

Comment:
Has this also been tested on the example submitted offline?

Reply:
yes.